src/porepy/params/parameter_dictionaries.py

```diff
@@ -15,106 +11,107 @@
+    d = {
+        "aperture": np.ones(g.num_cells),
+        "porosity": np.ones(g.num_cells),
+        "source": np.zeros(g.num_cells),
+        "mass_weight": np.ones(g.num_cells),
+        "second_order_tensor": pp.SecondOrderTensor(g.dim, np.ones(g.num_cells)),
+        "bc": pp.BoundaryCondition(g),
+        "bc_values": np.zeros(g.num_faces),
+        "darcy_flux": np.zeros(g.num_faces),
+        "time_step": 1,
+    }
+
+    # Ensure that parameters not handled above are copied. Values from in_data have
+    # priority over the default values.
     if not in_data:
         in_data = {}
-    d = in_data.copy()
-    # Ensure that the standard flow parameters are present in d. Values from in_data
-    # have priority over the default values.
-    d["aperture"] = in_data.get("aperture", np.ones(g.num_cells))
-    d["porosity"] = in_data.get("porosity", np.ones(g.num_cells))
-    d["source"] = in_data.get("source", np.zeros(g.num_cells))
-    d["time_step"] = in_data.get("time_step", 1)
-    # Set the second order tensor from the conductivity.
-    d["second_order_tensor"] = in_data.get(
-        "conductivity", pp.SecondOrderTensor(g.dim, np.ones(g.num_cells))
-    )
-    d["bc"] = in_data.get("bc", pp.BoundaryCondition(g))
-    d["bc_values"] = in_data.get("bc_values", np.zeros(g.num_faces))
-    d["darcy_flux"] = in_data.get(
-        "darcy_flux", pp.Upwind().darcy_flux(g, [0, 0, 0], d["aperture"])
-    )
-    d["mass_weight"] = in_data.get("mass_weight", d["porosity"])
+    d.update(in_data)
     return d
 
 
 def mechanics_dictionary(g, in_data=None):
     """ Dictionary with parameters for standard mechanics problems.
 
-    All parameters listed below which are not specified in in_data are assigned unitary
+    All parameters listed below which are not specified in in_data are assigned default
     values. Additional parameters may be passed in in_data.
-    Parameters:
+
+    Args:
         g: Grid.
         in_data: Dictionary containing any custom (non-default) parameter values.
-        kw: Keyword. The keyword is the identification connecting parameters and
-            discretizations.
+
     Returns:
         Dictionary with the "mathematical" parameters required by various flow
             discretization objects specified.
     """
-    # Ensure that parameters not handled below are copied or initialize empty dictionary
+    # Ensure that the standard mechanics parameters are present in d.
+    d = {
+        "aperture": np.ones(g.num_cells),
+        "porosity": np.ones(g.num_cells),
+        "source": np.zeros(g.dim * g.num_cells),
+        "mass_weight": np.ones(g.num_cells),
+        "fourth_order_tensor": pp.FourthOrderTensor(
+            g.dim, np.ones(g.num_cells), np.ones(g.num_cells)
+        ),
+        "bc": pp.BoundaryConditionVectorial(g),
+        "bc_values": np.zeros(g.dim * g.num_faces),
+        "slip_distance": np.zeros(g.num_faces * g.dim),
+        "time_step": 1,
```

Comment:
slip distance dies with the frictionSlipModel, and mechanics model. Again, maybe not this pullrequest